lib/rtoolsHCK.rb

```diff
@@ -1121,13 +1121,16 @@ def install_certificate_script(cer_path)
     ].join('; ')
   end
 
-  def install_driver_command(windows_path, install_method)
+  def install_driver_command(r_directory, windows_path, install_method)
     case install_method
     when 'PNP'
       "pnputil -i -a #{windows_path}"
     when 'NON-PNP'
       'RUNDLL32.EXE SETUPAPI.DLL,InstallHinfSection ' \
         "DefaultInstall 128 #{windows_path}"
+    when /^tapinstall-(.*)$/
```

Comment:
I don' like it.
I think it should be a generic option to run additional installer\configuration after installation.
Which one? I think it should be provided by the parameters and not be part of the code.